### PR TITLE
T naabus/textfixing

### PR DESCRIPTION
**Pull request by t-naabus**
Text Scrubbing

### CODE REVIEW

File: demo/code-demo/step_authorize.html [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+    <button id="nextStepWithSample" class="btn btn-primary" onclick="OpenEmbedStepWithSample()">Embed sample report</button><br/><br/><br/>
+    <a href="#" onclick="ToggleEmbedWithSpecificReportDiv()">Embed your own report</a><br/>
 </div>
 <div id="EmbedWithSpecificReportDiv" style="display: none;">
     <div class="line">
         <h4>Prerequisites</h4>
         <ul>
-            <li>A workspace in <a href="https://azure.microsoft.com/en-us/documentation/articles/power-bi-embedded-get-started/">Azure PowerBI Embedded Service.</a></li>
-            <li>PowerBI Report imported to your workspace.</li>
+            <li>A workspace in the <a href="https://azure.microsoft.com/en-us/documentation/articles/power-bi-embedded-get-started/">Power BI embedded service.</a></li>
```
Comment (noanu) [inline demo/code-demo/step_authorize.html]: you write Power BI (as separated words) above, and PowerBI as one word below.... Please make sure it is consistent.

File: demo/code-demo/step_authorize.html [hunk 2]
```diff
@@ -1,43 +1,44 @@
 <!DOCTYPE html>
 <div class="line">
-    <h4>Sample Report</h4>
-    To Embed using a sample report, you can just move to next step.<br/>
-    <button id="nextStepWithSample" class="btn btn-primary" onclick="OpenEmbedStepWithSample()">Embed Sample Report</button><br/><br/><br/>
-    <a href="#" onclick="ToggleEmbedWithSpecificReportDiv()">Embed Your Own Report</a><br/>
+    <h4>Sample report</h4>
+    To embed using a report supplied with the sample, select <b>Embed sample report</b>.<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: Embed Sample Report

this is a command not a sentence

File: demo/code-demo/report.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
         <div id="left-pane" class="halfWidth right">
             <div id="embedArea">
-                <h3>Embedded Report Area</h3>
+                <h3>Embedded report area</h3>
```
Comment (ranbreuer) [inline demo/code-demo/report.html]: Embedded Report Area

title

File: demo/code-demo/step_authorize.html (hunk 1 above)
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: Power BI

File: demo/code-demo/step_authorize.html [hunk 4]
```diff
@@ -1,43 +1,44 @@
...
+            <li>A workspace in the <a href="https://azure.microsoft.com/en-us/documentation/articles/power-bi-embedded-get-started/">Power BI embedded service.</a></li>
+            <li>PowerBI report imported to your workspace.</li>
         </ul>
     </div>
     <br/>
     <div class="line">
-        <h4>Flow</h4>
-        Once you have a report imported to PowerBI Workspace, you are ready to embed it.<br/>
-        To Embed a report you need to get an Embed-Token. You can create this token in multiple ways:<br/>
+        <h4>Instructions to get embed token</h4>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: ... generate embed App Token

File: demo/code-demo/step_authorize.html [hunk 5]
```diff
@@ -1,43 +1,44 @@
...
+        Once you have a report imported to Power BI workspace, you are ready to embed it.<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: Once you have improted a report into Power BI...

File: demo/code-demo/step_authorize.html [hunk 6]
```diff
@@ -1,43 +1,44 @@
...
+        <br/>
+        To embed a report, you need to get an embed-Token. You can create this token in multiple ways.<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: embed App Token

no hyphens
please be consistent

File: demo/code-demo/step_authorize.html [hunk 7]
```diff
@@ -1,43 +1,44 @@
...
+            <li>From NodeJS code using <a href="https://github.com/Microsoft/PowerBI-Node">powerbi-api</a> package.<br/></li>
         </ul>
     </div>
     <br/>
     <div class="line">
-        <h4>Enter Embed Details</h4>
+        <h4>Enter embed details</h4>
         
-        Enter Report Embed Token</br>
+        Enter report embed token<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: App Token

File: demo/code-demo/step_authorize.html [hunk 8]
```diff
@@ -1,43 +1,44 @@
...
         <input type="text" id="txtAccessToken" name="txtAccessToken" onchange="UpdateSessions();" style="width: 100%;" /><br/><br/>
         
-        Enter Report Embedded Url</br>
+        Enter report embedded url<br/>
         <input type="text" id="txtReportEmbed" name="txtReportEmbed" onchange="UpdateSessions();" style="width: 100%;" value="https://embedded.powerbi.com/appTokenReportEmbed" /><br/><br/>
         
-        Enter Report Id</br>
+        Enter report id<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: Id or ID

File: demo/code-demo/step_authorize.html [hunk 9]
```diff
@@ -1,43 +1,44 @@
...
     <div class="line">
-        <h4>Enter Embed Details</h4>
+        <h4>Enter embed details</h4>
         
-        Enter Report Embed Token</br>
+        Enter report embed token<br/>
         <input type="text" id="txtAccessToken" name="txtAccessToken" onchange="UpdateSessions();" style="width: 100%;" /><br/><br/>
         
-        Enter Report Embedded Url</br>
+        Enter report embedded url<br/>
```
Comment (ranbreuer) [inline demo/code-demo/step_authorize.html]: Url or URL

File: demo/code-demo/step_embed.html [hunk 10]
```diff
@@ -2,11 +2,11 @@
 <div id="settings">
     <table id="report-embed-table">
         <tr>
-            <td class="td-field-name">Embed Token</td>
+            <td class="td-field-name">Embed token</td>
```
Comment (ranbreuer) [inline demo/code-demo/step_embed.html]: App Token

File: demo/code-demo/step_authorize.html (hunk 1 above)
Comment (t-naabus) [inline demo/code-demo/step_authorize.html]: thanks, missed one

File: demo/code-demo/step_authorize.html (hunk 6 above)
Comment (t-naabus) [inline demo/code-demo/step_authorize.html]: just following what I was sent

**Pull request merged**